### DISCUSSION TITLE
Testing

### DISCUSSION THREAD

**Discussion by Meowcolm024** [Meowcolm024/FGO-Automata#52] [General]
日本語をテストします！
中文测试！

**Comment by Meowcolm024** [comment]
![shiki](https://fgo.wiki/images/e/ea/%E4%B8%A4%E4%BB%AA%E5%BC%8F%28Saber%29-%E5%A4%B4%E5%83%8F1.png)
Image testing :P

**Comment by Zhen-Bo** [comment]
有看到了,在發起討論中Orz

**Reply by Meowcolm024** [reply]
QWQ
